rebalance/rebalance.py

```diff
@@ -348,6 +348,11 @@ def a_minus_b(a: Millisatoshi, b: Millisatoshi):
     return a - b if a > b else Millisatoshi(0)
 
 
+def a_div_b(a: Millisatoshi, b: Millisatoshi):
+    # a divided by b, but don't divide by 0
+    return int(a) / int(b) if int(b) > 0 else 1
```

Comment:
This method is used to calculate the distortion of channel liquidity.
If `b` is set to `0` it means an extremely distorted scenario. 
Is returning `1` (neutral) a good approach here?

Also, in the code below `b` is given as `liquidity["ideal"]["our"]`, `...["their"]` and `liquidity["min"]`.
Should it be possible/normal these values can be `0`?
If not we can (and should) remove that check and replace it by a check in the calling function.

Also, also, if that check is removed, this function becomes pointless ;)

Also, also, also using recent pyln versions des not require int casts for multiplication and division, maybe we should change `requirements.txt` to recent version and cleanup the code where this also applies.

Reply:
In a distorted scenario `liquidity["ideal"]["their"]` can be zero. For example, if someone just started to run a node, opens some channels, and all liquidity belongs to him.

But yes, I will replace this with a check in the calling function.

Reply:
If they can be 0 legitimately you don't need to move that check to the caller.
Feel free to choose.